contentcuration/contentcuration/db/models/manager.py

```diff
@@ -47,6 +47,35 @@ def log_lock_time_spent(timespent):
     logging.debug("Spent {} seconds inside an mptt lock".format(timespent))
 
 
+# Fields that are allowed to be overridden on copies coming from a source that the user
+# does not have edit rights to.
+ALLOWED_OVERRIDES = {
+    "node_id",
+    "title",
+    "description",
+    "aggregator",
+    "provider",
+    "language_id",
+    "grade_levels",
+    "resource_types",
+    "learning_activities",
+    "accessibility_labels",
+    "categories",
+    "learner_needs",
+    "role",
+    "extra_fields",
+    "suggested_duration",
+}
+
+EDIT_ALLOWED_OVERRIDES = ALLOWED_OVERRIDES.union({
```

Comment:
With these 3 words all in the variable name, the variable's purpose took me a moment to decipher (thinking 'overrides' already feels like an 'edit'). Not sure if `EDITOR` makes it clearer?

Reply:
Yeah, I was not at the race for my variable naming this day. `EDITOR` does make a bit more sense, definitely.